src/elm/Logs.elm

```diff
@@ -469,3 +493,53 @@ toString log =
 
         Nothing ->
             ""
+
+
+
+-- ANSI
+
+
+{-| defaultLogModel : struct to represent default model required by ANSI parser
+-}
+defaultLogModel : Ansi.Log.Model
```

Comment:
models required by the ansi parser package
see:
https://package.elm-lang.org/packages/vito/elm-ansi/latest/Ansi-Log